static_report/src/components/SingleReport/SRBarChart.tsx

```diff
@@ -1,33 +0,0 @@
-import { Flex } from '@chakra-ui/react';
-import { useRef } from 'react';
-import { useResizeObserver } from '../../hooks/useResizeObserver';
-import { useSingleChart } from '../../hooks/useSingleChart';
-import { ColumnSchema } from '../../sdlc/single-report-schema';
-
-export interface BarChartDatum {
-  type: ColumnSchema['type'];
-  isCategorical: boolean;
-  label: string | null;
-  value: any;
-  total: number;
-}
-interface Props {
-  data: Array<BarChartDatum>;
-}
-
-export function SRBarChart({ data }: Props) {
```

Comment:
For the single bar chart, I still use it in my [branch](https://github.com/InfuseAI/piperider/tree/feature/sc-27965/new-layout-single-run-overview-page) for the new overview design.

My opinion is to keep the file. After the PR merge, I will remove it because I need to migrate the chart (d3) to the new chart (chart.js).

Reply:
Do you need `CRBarChart` as well?

Reply:
yes, thank you.

Reply:
will revert single chart in a moment

Reply:
done